crates/bevy_app/src/sub_app.rs

```diff
@@ -219,6 +222,17 @@ impl SubApp {
         self
     }
 
+    /// See [`App::remove_systems_in_set`]
+    pub fn remove_systems_in_set<M>(
```

Comment:
I'm curious about the intended use cases here, given the overhead tradeoffs. Whats a real world scenario where someone would want to call this?

Reply:
> I'm curious about the intended use cases here, given the overhead tradeoffs. Whats a real world scenario where someone would want to call this?

I'm not sure what others are planning, but I've been tinkering with building a plugin to define non-exclusive systems with scripting languages.  It would want to remove a system if its script was removed, or if the parameters to its script changed and it needed to be replaced with a system with different parameters.  Loading the new script wouldn't be very performance-sensitive.

Reply:
I also think that we might be able to use this for hotpatching!